pcsx2-qt/Settings/MemoryCardConvertDialog.cpp

```diff
@@ -30,6 +31,8 @@ MemoryCardConvertDialog::MemoryCardConvertDialog(QWidget* parent, QString select
 	: QDialog(parent)
 {
 	m_ui.setupUi(this);
+	MouseWheelFilter* wf = new MouseWheelFilter(this);
```

Comment:
`MemoryCardConvertDialog::setupUi` is defined inside a header file, therefor I don't want to create an object inside if it.

Reply:
Unless it's fine.

Also in the debugger it must be called after the addTab() functions, so can't do it in setupUi unless they are incorporated in the function as well.
https://github.com/PCSX2/pcsx2/blob/b55b0482384d71b4fa143e4d31c21dd8b3673af9/pcsx2-qt/Debugger/DebuggerWindow.cpp#L64-L66